src/App/helpers/jwt.js

```diff
@@ -0,0 +1,28 @@
+import 'dotenv/config';
+import { sign, verify } from 'jsonwebtoken';
+import { compare, hash } from 'bcryptjs';
+
+const { JWT_SECRET, JWT_EXPIRE } = process.env;
+
+class encryption {
+static async encryptPassword(password) {
```

Comment:
Why do you have this helper to encrypt twice as in `helpers/encryption.js` and here?